iree/compiler/Dialect/Flow/IR/PartitionableLoopsInterface.h

```diff
@@ -0,0 +1,18 @@
+// Copyright 2021 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#ifndef IREE_COMPILER_DIALECT_FLOW_IR_PARTITIONABLE_LOOPS_INTERFACE_H_
```

Comment:
Should this be in an `Interfaces` directory? It has nothing to do with `Flow/IR` actually; it's just attaching foreign models to ops from various other dialects.